src/main.cpp

```diff
@@ -7,6 +7,7 @@
 #include "main/dissolve.h"
 #include "main/version.h"
 #include "math/mathFunc.h"
+#include <format>
```

Comment:
A pernickety blank line request!
```suggestion
#include <format>

```